library/psa_crypto_core.h

```diff
@@ -607,4 +607,303 @@ psa_status_t psa_key_agreement_raw_builtin(
     size_t shared_secret_size,
     size_t *shared_secret_length );
 
+/**
+ * \brief Set the maximum number of ops allowed to be executed by an
+ *        interruptible function in a single call.
+ *
+ * \warning This is a beta API, and thus subject to change at any point. It is
+ *          not bound by the usual interface stability promises.
+ *
+ * \note The signature of this function is that of a PSA driver
+ *       interruptible_set_max_ops entry point. This function behaves as an
+ *       interruptible_set_max_ops entry point as defined in the PSA driver
+ *       interface specification for transparent drivers.
+ *
+ * \param[in]  max_ops          The maximum number of ops to be executed in a
+ *                              single call, this can be a number from 0 to
+ *                              #PSA_INTERRUPTIBLE_MAX_OPS_UNLIMITED, where 0
+ *                              is obviously the least amount of work done per
+ *                              call.
+ */
+void mbedtls_psa_interruptible_set_max_ops( uint32_t max_ops );
```

Comment:
https://github.com/Mbed-TLS/mbedtls/issues/7032

Reply:
When you say "allowing drivers to call `get_max_ops()`", are the drivers able to just call the PSA interface, or do we need something else?

Reply:
Drivers would call the PSA API function. The driver specification should explicitly state that this is allowed, since [API functions are not guaranteed to be reentrant in general](https://github.com/Mbed-TLS/mbedtls/issues/7043).